src/widgets/dcombobox.cpp

```diff
@@ -45,7 +45,8 @@ void DComboBoxPrivate::init()
 }
 
 /*!
- * \~chinese \brief 获得控件屏幕矩形位置，QDesktopWidgetPrivate改为QApplication::desktop()访问应用位置。
+ * @~english
+    @brief Obtain the screen rectangle position of the control, replace QDesktopWidgetPrivate with QApplication::desktop() to access the application position.
```

Comment:
Obtain the screen rectangle of the widget. Use QScreen to access the screen geometry.